src/Chapter15/Listing15.27.ProjectionToAnAnonymousType.cs

```diff
@@ -36,4 +40,6 @@ private static void Print<T>(IEnumerable<T> items)
             }
         }
     }
+    #endregion EXCLUDE
+    #endregion INCLUDE
```

Comment:
Would it be simpler to just move the #region INCLUDE to just surround the inner section instead of the #exclude right next to an include?